app/src/main/java/com/example/pix/home/fragments/ProfileFragment.java

```diff
@@ -136,11 +157,18 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         } else {
             // The friend's name is a TextView, so that the user cannot edit it
             TextView name = view.findViewById(R.id.profile_name);
-            name.setText("" + user.getUsername());// If we press "Enter" in the username EditText, update the username
+            name.setText(user.getUsername());// If we press "Enter" in the username EditText, update the username
         }
         // Set User's number of Pix
         TextView pix = view.findViewById(R.id.profile_pix);
-        pix.setText("" + user.getInt(USER_PIX));
+        // Check every 2 seconds for how many "Pix"(Likes) this person has
+        timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                getActivity().runOnUiThread(() -> pix.setText(Like.getPix(user) + "P"));
+            }
+        }, 0, 2000);
 
```

Comment:
I'm curious, is this really important such that you need to refresh all the time to get the latest likes?

Reply:
Would it be better to do less often, or that I don't do it at all, just once when starting the Fragment?

Reply:
If it's not that important for it to be updated all the time, I'd suggest not having this timer.